### DISCUSSION TITLE
Warp extension

### DISCUSSION THREAD

**Discussion by WolfgangHG** [arquillian/arquillian-core#455] [General]
What is the state of the Warp extension? It needs a bit of work to run on Java 11 and beyond ([https://issues.redhat.com/browse/ARQ-2216](https://issues.redhat.com/browse/ARQ-2216)). 
And there are probably major changes necessary to make it JakartaEE10 compatible.

I could try to make the changes, but I will probably help and guidance, as I don't have deeper knowledge of Arquillian - I have just a small sample that uses Warp.

**Comment by chengfang** [comment]
This discussion has moved to https://github.com/arquillian/arquillian-extension-warp where various issues and PRs have been worked on.
